src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,334 @@
+
+const styles = (theme: CustomTheme) => ({
+  contentContainer: {
+    marginTop: '1rem',
+    maxWidth: 1000,
+  },
+  form: {
+    width: '100%',
+  },
+  formFieldContainer: {
+    minHeight: 135,
+  },
+  formFieldDescription: {
+    marginBottom: 10,
+  },
+  formLabel: {
+    display: 'block',
+    fontWeight: 'bold',
+    marginTop: 10,
+    marginBottom: 10,
+  },
+  formLabelError: {
+    color: theme.palette.error.main,
+  },
+  formInput: {
+    width: '100%',
+  },
+  formInputError: {
+    color: theme.palette.error.main,
+    fontSize: '0.75rem',
+    lineHeight: '1.66',
+    marginLeft: 14,
+  },
+});
+
+interface IProps extends WithStyles<typeof styles> {
+  userEmail: string;
+}
+
+const DatasetSchemaInformationView = withStyles(styles)(({ classes }: IProps) => {
+  const [regionOptions, setRegionOptions] = useState(GCP_REGIONS);
+  const {
+    register,
+    control,
+    formState: { errors },
+  } = useFormContext();
+
+  const editorOptions = useMemo(
+    () =>
+      ({
+        previewRender(markdownText) {
+          return ReactDOMServer.renderToString(
+            <WithoutStylesMarkdownContent markdownText={markdownText} />,
+          );
+        },
+        status: false,
+      } as SimpleMDE.Options),
+    [],
+  );
+
+  return (
+    <Grid container rowSpacing={2} columnSpacing={5} className={classes.contentContainer}>
+      <Grid item xs={12}>
+        <Typography variant="h3">Dataset information</Typography>
+        We need some basic information about the dataset
+      </Grid>
+      <Grid item xs={12} md={6} lg={6} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-name"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.name })}
+        >
+          Name your dataset*
+        </label>
+        <TextField
+          id="dataset-name"
+          className={classes.formInput}
+          placeholder="Dataset name"
+          variant="outlined"
+          error={!!errors.name}
+          helperText={errors.name ? errors.name.message : ''}
+          {...register('name', {
+            required: 'name is required',
+            minLength: { value: 1, message: 'Name must be 1+ characters long' },
+            maxLength: { value: 511, message: 'Name must be less than 511 characters long' },
+            pattern: {
+              value: /^[a-zA-Z0-9][_a-zA-Z0-9]*$/,
+              message: 'Name should fit pattern: ^[a-zA-Z0-9][_a-zA-Z0-9]*$',
+            },
+          })}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-description"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.description })}
+        >
+          Dataset description*
+        </label>
+        <Controller
+          name="description"
+          control={control}
+          rules={{ required: 'description is required' }}
```

Comment:
Should there be a max length for this?

Reply:
Oh shoot, I didnt notice there was a maxlen for this in the mock up!!! Ok, fixing!